tests/lib/assertions/developer1-2410-classic-dangerous.json

```diff
@@ -0,0 +1,42 @@
+{
+    "type": "model",
+    "authority-id": "developer1",
+    "series": "16",
+    "brand-id": "developer1",
+    "model": "developer1-2410-classic-dangerous",
+    "architecture": "amd64",
+    "timestamp": "2024-04-09T22:00:00+00:00",
+    "grade": "dangerous",
+    "base": "core22",
```

Comment:
We probably want core24 here. Or is it maybe necessary because the kernel in `classic-24.10/edge` uses core24 incorrectly?

Reply:
AFAIK, this is aimed at classic-25.04, which seems to use core22 as its base https://github.com/canonical/models/blob/master/ubuntu-classic-2504-amd64.json. I was mostly trying to mimic the target system. I don't mind switching to core24 though.

Reply:
Maybe worth asking in the next sync, the target should be core24 eventually.